blocks/components/editable/style.scss

```diff
@@ -69,3 +87,44 @@ input.editable-format-toolbar__link-input {
 	padding: 10px;
 	flex-grow: 1;
 }
+
+/* Block Placeholders */
+.editor-visual-editor__block .is-placeholder {
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+	justify-content: center;
+	padding: 1em;
+	min-height: 200px;
+	text-align: center;
+	font-family: $default-font;
+	font-size: $default-font-size;
+	background: $light-gray-100;
+}
+
+.placeholder__label {
```

Comment:
What's the plan for this? Is this going to become a general-purpose component? We're breaking our own self-imposed CSS naming guidelines with this change:

https://github.com/WordPress/gutenberg/blob/master/docs/coding-guidelines.md#naming